internal/services/costmanagement/export_resource_base.go

```diff
@@ -29,6 +29,14 @@ func (br costManagementExportBaseResource) arguments(fields map[string]*pluginsd
 			Default:  true,
 		},
 
+		"partition_data_enabled": {
+			Type:     pluginsdk.TypeBool,
+			Optional: true,
+			Required: false,
```

Comment:
We don't need to specify `false` for `Required` since it's implied by `Optional` being set to `true`
```suggestion
```